code/lib/cli/src/generators/SVELTE/index.ts

```diff
@@ -1,43 +1,13 @@
-import fse from 'fs-extra';
-import { logger } from '@storybook/node-logger';
-
 import { CoreBuilder } from '../../project_types';
 import { baseGenerator } from '../baseGenerator';
 import { Generator } from '../types';
 
 const generator: Generator = async (packageManager, npmOptions, options) => {
-  let extraMain;
-  // svelte.config.js ?
-  if (fse.existsSync('./svelte.config.js')) {
-    logger.info("Configuring preprocessor from 'svelte.config.js'");
-
-    extraMain = {
-      svelteOptions: { preprocess: '%%require("../svelte.config.js").preprocess%%' },
-    };
-  } else if (fse.existsSync('./svelte.config.cjs')) {
-    logger.info("Configuring preprocessor from 'svelte.config.cjs'");
-
-    extraMain = {
-      svelteOptions: { preprocess: '%%require("../svelte.config.cjs").preprocess%%' },
-    };
-  } else {
-    // svelte-preprocess dependencies ?
-    const packageJson = packageManager.retrievePackageJson();
-    if (packageJson.devDependencies && packageJson.devDependencies['svelte-preprocess']) {
-      logger.info("Configuring preprocessor with 'svelte-preprocess'");
-
-      extraMain = {
-        svelteOptions: { preprocess: '%%require("svelte-preprocess")()%%' },
-      };
-    }
-  }
-
```

Comment:
as per https://github.com/storybookjs/storybook/issues/19280#issuecomment-1264132087

Reply:
Does this need any MIGRATION.md notes? @JReinhold

Reply:
You're right, users needs to remove their `svelteOptions` property in `main.cjs` and rely on their Vite config to pick them up instead. Svelte+Webpack users probably don't want to remove anything.

I'll write it up.

Reply:
What do you think about pulling this change out to its own PR?  I'd love to get it into the next release, since svelte apps are currently broken after bootstrapping with Storybook.